src/iterators/mod.rs

```diff
@@ -726,6 +726,18 @@ where
     }
 }
 
+impl<'a, A, D> DoubleEndedIterator for LanesIter<'a, A, D>
+where
+    D: Dimension,
```

Comment:
I expected this to be limited to the case where D=Ix1. Just like how .iter()'s `Iter` works. I think it's better that way, then the Baseiter does not leak out (it's in the where clause, that's a leak into public API.)

Reply:
Fixed that, and also applied the same change to LanesIterMut - it and LanesIter must of course change together.